pipelines/controller/config/profile-definitions/StructureDefinition-us-core-birthsex.json

```diff
@@ -0,0 +1,327 @@
+{
+  "resourceType": "StructureDefinition",
```

Comment:
If we package all these definitions in one module and create a jar out of it, then the structure definition files have to be read as `classpath` files, whereas currently the `ProfileMapperFhirContexts.contextFor(FhirVersionEnum fhirVersion, @Nullable String structureDefinitionsPath)` function expects files from a directory instead of classpath. This is done this way so that the user can configure definitions in a directory during deployment.

Reply:
provided classpath support and have removed the definitions from the `pipelines`.